### DISCUSSION TITLE
How can I merge system audio and microphone inputs into a single output and stream them to an Ant Media Server?

### DISCUSSION THREAD

**Discussion by tremzo** [ant-media/Ant-Media-Server#5760] [FAQs]
How can I merge system audio and microphone inputs into a single output and stream them to an Ant Media Server?

**Comment by lastpeony** [comment]
Users can merge both system audio and a microphone input, allowing them to stream a unified audio output to Ant Media Server using the free VoiceMeeter software on Windows.

1-) Download and install voice meeter from this link https://vb-audio.com/Voicemeeter/
2-) Choose HARDWARE INPUT 1 as your desired microphone device.
3-) Leave HARDWARE INPUT 2 empty.
4-) Open windows sound settings and choose default device as VoiceMeeter Input. You wont be able to hear system audio now on but dont worry.
![image](https://github.com/ant-media/Ant-Media-Server/assets/24750845/c6b31b57-da42-4c91-8607-53254d8e68c6)
5-) Choose HARDWARE OUT as your computers speaker.
6-) Your VIRTUAL INPUT should be VB-Audio Voicemeeter VAIO because we selected it as default device at step 4.
7-) Uncheck A circle of HARDWARE INPUT 1 or you will hear your own voice.
Before we start streaming your voice meeter should look like this:
![image](https://github.com/ant-media/Ant-Media-Server/assets/24750845/994422c3-6d25-487d-9ffc-8ab28ff1e111)

Now go to ant media publish page and choose your audio source as VoiceMeeter Output (VB-Audio VoiceMeeter VAIO)
![image](https://github.com/ant-media/Ant-Media-Server/assets/24750845/dadce62e-a0cb-4cac-880f-39c3e4186f39)

Start publish.

When you play the stream you will hear system audio+microphone.
